addons/website_event_migration_layer/views/website_event_booth.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<!-- ========================================================== -->
+<!-- addons/website_event_booth/views/event_booth_templates.xml -->
+<!-- ========================================================== -->
+<template id="event_booth_layout__ml" inherit_id="website_event_booth.event_booth_layout" name="event_booth_layout__ml">
+    <xpath expr="//h3" position="replace">
+        <attribute name="class" add="mb-4" remove="mb-2" separator=" "/>
+    </xpath>
+</template>
+
+<template id="event_booth_registration__ml" inherit_id="website_event_booth.event_booth_registration" name="event_booth_registration__ml">
+    <xpath expr="//t[@t-call='website_event_booth.event_booth_order_progress']" position="replace">
+        <div class="oe_structure"/>
+        <div class="d-flex flex-wrap align-items-center justify-content-between my-3">
+            <h3 class="my-0">Get A Booth</h3>
+            <t t-call="website_event_booth.event_booth_order_progress">
+                <t t-set="step" t-value="'STEP_BOOTH_SELECTION'"/>
+            </t>
+        </div>
+        <div class="oe_structure"/>
+    </xpath>
+    <xpath expr="//h4" position="replace">
+        <h5 class="mt-0 mb-3">Choose your type of booth</h5>
+    </xpath>
+
+    <!-- Booth Card -->
+    <xpath expr="//t[@t-as='booth_category']/div" position="attributes">
+        <attribute name="class"/>
+        <attribute name="t-attf-class">col-md-6 col-lg-4 mb-4 {{ (len(event.event_booth_category_ids) &gt; 3) and 'col-xxl-3' }}</attribute>
+    </xpath>
+    <xpath expr="//t[@t-as='booth_category']//label" position="attributes">
+        <attribute name="t-attf-class"/>
+        <attribute name="t-attf-class">d-block h-100 #{'o_wbooth_category_unavailable overflow-hidden' if booth_category_unavailable else ''}"</attribute>
+    </xpath>
+    <xpath expr="//t[@t-as='booth_category']//div[hasclass('card-body')]" position="attributes">
+        <attribute name="class" add="d-flex flex-wrap w-100 gap-2 justify-content-between flex-grow-0 pb-0" separator=" "/>
+    </xpath>
+    <xpath expr="//t[@t-as='booth_category']//h5" position="attributes">
+        <attribute name="class" add="my-0" separator=" "/>
+    </xpath>
+    <xpath expr="//t[@t-as='booth_category']//div[@t-field='booth_category.description']" position="replace"/>
+    <xpath expr="//t[@t-as='booth_category']//div[hasclass('card-body')]" position="after">
+        <div class="w-100 small" t-attf-id="o_wbooth_booth_description_#{booth_category.id}" t-field="booth_category.description"/>
+    </xpath>
```

Comment:
Really no other way to do it than by changing DOM ?

Reply:
Unfortunately not :/